src/blocks/gallery-masonry/controls.js

```diff
@@ -1,28 +0,0 @@
-/**
```

Comment:
Why is this removed?

Reply:
Media Filters were applied ad-hoc to the custom inner images that we used in the galleries. We apply the Media Filters by Extension to the core/image block so we already have those controls on the images thus we do not need them here.

Reply:
Ok, thanks for the explanation!

Reply:
For posterity, I ended up re-adding the media-filter controls the masonry block with some refactors to extension logic. We now have a hierarchy of controls.